tests/ci/api_run.sh

```diff
@@ -28,10 +28,10 @@ function uploader {
 set +e
 
 docker ps
-
+echo `df -H`
```

Comment:
Could you add more context or remove the `df` which is mainly for debug purpose?